templates/semantic-ui/invenio_notifications/comment-request-event.create.jinja

```diff
@@ -45,6 +53,8 @@ Best regards,
 --
 CERN Document Server <{{ config.SITE_UI_URL }}>
 Need help? <{{ help_url }}>
+
+{{ _("This is an auto-generated message. To manage notifications, visit your account settings")}}
```

Comment:
Normally we don't add translation, as the `notification` template already supports translations in the path, you can provide a new jinja template per language